atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -298,9 +299,13 @@ public static String internalTableName(TableReference tableRef) {
         return tableName.replaceFirst("\\.", "__");
     }
 
+    /** @deprecated uses TableReference.createUnsafe, which is itself deprecated.
+     *
+     */
+    @Deprecated
```

Comment:
I do plan on removing that, but in a future release. I want to give certain internal products a grace period.